homeassistant/components/envisalink.py

```diff
@@ -149,7 +154,7 @@ def stop_envisalink(event):
 
     def start_envisalink(event):
         """Startup process for the Envisalink."""
-        EVL_CONTROLLER.start()
+        hass.loop.run_in_executor(None, EVL_CONTROLLER.start)
```

Comment:
The problem is that your start method doesn't handle asyncio correctly. Some things to fix:

- You're calling `asyncio.async`, a deprecated method (now called ensure_future). [docs](https://docs.python.org/3/library/asyncio-task.html#asyncio.ensure_future)
- You're not passing the eventloop into `async` so it uses the default eventloop